test/cypress/support/index.js

```diff
@@ -78,4 +78,12 @@ Cypress.Commands.add('addIndicator', () =>
         .children('.highcharts-popup button')
         .eq(0)
         .click()
-);
\ No newline at end of file
+);
+
+Cypress.Commands.add('selectAnnotation', (annotationClassName, parentClassName) => {
```

Comment:
I was wondering if there should be some sort of comment/documentation explaining each parameter?
On the other hand, the names are pretty self-explanatory...
But it's just a thought, so it's up to you ;)